Sources/Confidence/ConfidenceValue.swift

```diff
@@ -0,0 +1,231 @@
+import Foundation
+
+public typealias ConfidenceStruct = [String: ConfidenceValue]
+
+public class ConfidenceValue: Equatable, Encodable {
+    private let value: ConfidenceValueInternal
+
+    public init(boolean: Bool) {
+        self.value = .boolean(boolean)
+    }
+
+    public init(string: String) {
+        self.value = .string(string)
+    }
+
+    public init(integer: Int64) {
+        self.value = .integer(integer)
+    }
+
+    public init(double: Double) {
+        self.value = .double(double)
+    }
+
+    public init(date: DateComponents) {
```

Comment:
I am unsure that `DateComponents` is the best Swift type to express a calendar day